input/fsh/GGGenomicStudy.fsh

```diff
@@ -72,23 +72,33 @@ Description: "A genomic study is a set of analyses performed to analyze and gene
 * usedReference 0..0
 * usedCode 0..0
 
+Extension:   GenomicStudyAnalysisGenomicSourceClass
+Id:          genomic-study-analysis-genomic-source-class
+Title:      "Genomic Study Analysis Method Type"
+Description: "The genomic class of the specimen being analyzed: Germline for inherited genome, somatic for cancer genome, and prenatal for fetal genome."
+* ^context[+].type = #element
+* ^context[=].expression = "Procedure"
+* value[x] only CodeableConcept
+* valueCodeableConcept from http://loinc.org/vs/LL378-1 (extensible)
+
 Extension:   GenomicStudyAnalysisMethodType
 Id:          genomic-study-analysis-method-type
 Title:      "Genomic Study Analysis Method Type"
 Description: "Defines a method type for a genomic analysis"
 * ^context[+].type = #element
 * ^context[=].expression = "Procedure"
 * value[x] only CodeableConcept
-* valueCodeableConcept from GenomicStudyMethodTypeVS (example)
+* valueCodeableConcept from GenomicStudyMethodTypeVS (preferred)
```

Comment:
Simliar to line 100, should we add some example codes to ^short attribute to this method type?

Reply:
added